app/models/host_storage.rb

```diff
@@ -2,6 +2,9 @@ class HostStorage < ApplicationRecord
   belongs_to :host
   belongs_to :storage
 
+  supports(:update) { _("Can not edit read only model") if read_only? }
+  supports(:delete) { _("Can not delete read only model") if read_only? }
+
```

Comment:
This means that the _storage_ is read only from the POV of the host, not that the _record_ is read-only.

Reply:
thnx